lib/Doctrine/Common/Persistence/Mapping/Driver/SymfonyFileLocator.php

```diff
@@ -54,15 +54,24 @@ class SymfonyFileLocator implements FileLocator
     protected $fileExtension;
 
     /**
+     * Represents PHP namespace delimiters when looking for files
+     *
+     * @var string
+     */
+    protected $nsSeparator;
+
+    /**
      * Constructor.
      *
      * @param array       $prefixes
      * @param string|null $fileExtension
+     * @param string      $nsSeparator
```

Comment:
Is class property documentation not enough? Or should I move it here or duplicate?

```php
    /**
     * Represents PHP namespace delimiters when looking for files
     *
     * @var string
     */
    protected $nsSeparator;
```